docs/topics_5/proc_web-troubleshooting-common-issues.adoc

```diff
@@ -0,0 +1,55 @@
+= Common issues when troubleshooting a {WebName} installation on OpenShift Container Platform
+
+.`NoRouteToHostException` error
+
+The following error might appear in the `mta-web-console-executor` pod log:
+
+----
+13:44:03,501 SEVERE [org.jboss.windup.web.messaging.executor.ExecutorBootstrap] (main) Could not start messaging listener due to: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]: javax.naming.CommunicationException: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]
+----
+
+This error message is displayed in a new deployment because the `mta-web-console-executor` pod starts before the `mta-web-console` pod. Wait for the `mta-web-console` pod to start.
+
+.A pod reports insufficient resources
+
+After attempting to deploy {WebNameTitle} on OpenShift, a pod is unable to start, and the following error is seen in the `Events` tab:
+
+[source,options="nowrap"]
+----
+0/9 nodes are available: 4 Insufficient cpu, 4 MatchNodeSelector, 9 Insufficient memory.
+----
+
+The resource quota for the OpenShift project has been met and the pod is unable to obtain the requested resources.
+
+.Procedure
+
+. Do either of the following:
+
+* Increase the quota for the OpenShift project. For additional information on OpenShift quotas, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges].
+* Reduce the requested resources for the {WebName} OpenShift project. It is recommended to have at least 2 CPUs and 4 GB of memory for the project.
+
+. Once the request is within the available quota, attempt the deployment once again.
+
+.Deployment pods time out
+
+The `mta-web-console-deploy` and `mta-web-console-executor-deploy` pods time out and the following error is displayed in their logs:
+
+[source,options="nowrap",subs="+quotes"]
+----
+error: update acceptor rejected mta-web-console-executor-1: Pods for rc 'mta/<POD_NAME>' took longer than 600 seconds to become available
+----
+
+This error might be caused by insufficient cluster resources or unsuccessful attempts to pull images from the registry.
+
+.Procedure
+
+. Redeploy the {WebName} and observe the `mta-web-console`, `mta-web-console-executor`, and `mta-web-console-postgresql` pod logs during the deployment for errors.
+
+* If the OpenShift instance is out of resources, follow the instructions in link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[Analyzing Cluster Capacity] to determine the cluster capacity. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment again.
+
+* If the images cannot be pulled from the registry, link:{OpenShiftDocsURL}/registry/accessing-the-registry.html[check the registry].
```

Comment:
I thought Phil wrote in the Jira comment that an image pull failure almost never happens.